hugr-py/src/hugr/_hugr.py

```diff
@@ -292,6 +292,7 @@ def _node_links(
             return
         # iterate over known offsets
         for offset in range(self.num_ports(node, direction)):
+            # TODO should this also look for -1 state order edges?
```

Comment:
I'd prefer a second method to get the order links

Reply:
just the order links or the order links along with the value?